stable/insights-agent/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
 description: A Helm chart to run the Fairwinds Insights agent
 name: insights-agent
-version: 0.13.0
-appVersion: 0.6.0
+version: 0.13.1
```

Comment:
My best guess :) I figured since 0.13 never got released with a version of Insights it seemed safe to re-use that minor version. I was going back and forth on it though so I'd easily bump the minor version.

Reply:
Ah yeah just saw your comment above. I'd say minor just to keep it semver-y, but no big deal since Insights still recommends `0.12.*`